src/iosrv/server_iv.c

```diff
@@ -974,6 +1049,7 @@ iv_op(struct ds_iv_ns *ns, struct ds_iv_key *key, d_sg_list_t *value,
 		arg->shortcut = shortcut;
 		arg->iv_sync = *sync;
 		arg->retry = retry;
+		ds_iv_ns_get(ns);
```

Comment:
Is it guaranteed sync_comp_cb() must be called to put the reference? It looks not that obvious to me, maybe the refcount get/put places needs be revised a bit to make it obvious.

Reply:
well, ns is bonded with argument, and its reference will be put when the argument is freed anyway.